src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -128,12 +158,23 @@ def delete_btrfs_subvolume(devicegraph, path)
       #
       # @param path [string] absolute subvolume path
       # @param nocow [Boolean] no copy on write property
+      #
+      # @return [BtrfsSubvolume, nil] new subvolume
+      #
       def create_btrfs_subvolume(path, nocow)
-        subvolume = default_btrfs_subvolume.create_btrfs_subvolume(path)
-        subvolume.nocow = nocow
-        subvolume_mount_path = btrfs_subvolume_mount_point(path)
-        subvolume.create_mount_point(subvolume_mount_path) unless subvolume_mount_path.nil?
-        subvolume
+        subvolume = find_btrfs_subvolume_by_path(path)
```

Comment:
So now there is a chance that `#create_btrfs_subvolume` returns a subvolume that it didn't actually create. Please, document that. It's completely unexpected to me.